pkg/cmd/server/server.go

```diff
@@ -832,3 +826,39 @@ func (s *server) runProfiler() {
 		s.logger.WithError(errors.WithStack(err)).Error("error running profiler http server")
 	}
 }
+
+// runLiveness starts an HTTP server to handle liveness checks for the velero server.
+// Note that liveness checks should have generous timeouts, since Velero will query
+// object storage, which is possibly a network call across the internet.
+func (s *server) runLiveness() {
+	mux := http.NewServeMux()
+	mux.HandleFunc("/livez", func(w http.ResponseWriter, r *http.Request) {
+		// Refresh the discovery helper, since it may be stale if the velero API group was
+		// created after the pod.
+		s.discoveryHelper.Refresh()
```

Comment:
I'm not sure the `Refresh` call belongs here - it may be better inside of `veleroResourcesExist`.

Reply:
somewhat related, we already have a goroutine that refreshes discovery every 5min -- so we'll end up refrehsing a lot. Not sure yet where it makes sense to have it, just food for thought right now.

Reply:
Yeah; I added it because it wasn't picking up the CRDs even after I posted them. 5 minutes seems like a long time for a liveness check, but maybe we remove this refresh and change the other one to match whatever the check period is on the pod?